go.mod

```diff
@@ -175,8 +162,9 @@ require (
 	golang.org/x/mod v0.4.2 // indirect
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c // indirect
 	golang.org/x/text v0.3.7 // indirect
-	golang.org/x/tools v0.1.2 // indirect
+	golang.org/x/tools v0.1.5 // indirect
 	google.golang.org/genproto v0.0.0-20210602131652-f16073e35f0c // indirect
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+	lukechampine.com/blake3 v1.1.6 // indirect
```

Comment:
Not going to actually complain, but I would very much like to know why and where Blake3 is being used 😅

Reply:
```
$ go mod why lukechampine.com/blake3
...
github.com/ipfs/go-cid
github.com/multiformats/go-multihash
github.com/multiformats/go-multihash/register/all
github.com/multiformats/go-multihash/register/blake3
```